data/modules/SearchRescue/SearchRescue.lua

```diff
@@ -718,8 +718,8 @@ local createTargetShip = function (mission)
 	elseif mission.flavour.loctype == "CLOSE_SPACE" then
 		ship = Space.SpawnShipNear(shipdef.id, Space.GetBody(mission.station_target.bodyIndex), mission.dist/1000, mission.dist/1000)
 	elseif mission.flavour.loctype == "FAR_SPACE" then
-		ship = Space.SpawnShipNear(shipdef.id, Space.GetBody(mission.planet_target.bodyIndex), far_space_orbit_dist, far_space_orbit_dist)
-		ship:AIEnterHighOrbit(Space.GetBody(mission.planet_target.bodyIndex))
+		local orbit_radius = Space.GetBody(mission.planet_target.bodyIndex):GetPhysicalRadius() * far_space_orbit_dist
+		ship = Space.SpawnShipOrbit(shipdef.id, Space.GetBody(mission.planet_target.bodyIndex), orbit_radius, orbit_radius)
```

Comment:
I would recommend this cache the result of Space.GetBody instead, as that's a more expensive call than calculating the orbit radius.
```suggestion
		local target_body = Space.GetBody(mission.planet_target.bodyIndex)
		local orbit_radius = target_body:GetPhysicalRadius() * far_space_orbit_dist
		ship = Space.SpawnShipOrbit(shipdef.id, target_body, orbit_radius, orbit_radius)
```